### PR TITLE
antlr4-cpp-runtime@4.12.0

### PR DESCRIPTION
**Pull request by mmorel-35**
Release: https://github.com/antlr/antlr4/releases/tag/4.12.0

Used by:
* https://github.com/google/fuzztest

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (antlr4-cpp) have been updated in this PR. Please review the changes.

File: modules/antlr4-cpp/4.12.0/MODULE.bazel [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module(
+    name = "antlr4-cpp",
```
Comment (fmeum) [inline modules/antlr4-cpp/4.12.0/MODULE.bazel:2]: Does it make sense to include the version number in the name? (Not saying it doesn't, just want to start a discussion on the potential duplication).

Comment (fmeum) [reply modules/antlr4-cpp/4.12.0/MODULE.bazel:2]: Sounds good, but shouldn't we also give the target the same name then? Or at least add an alias.

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (antlr4-cpp-runtime) have been updated in this PR. Please review the changes.

**Review by fmeum — APPROVED** [review]

**Review by fmeum — APPROVED** [review]

**Pull request merged**